src/community/notification-common/pom.xml

```diff
@@ -20,7 +20,7 @@
 		<dependency>
 			<groupId>com.rabbitmq</groupId>
 			<artifactId>amqp-client</artifactId>
-			<version>4.0.0</version>
+			<version>5.9.0</version>
```

Comment:
@afabiani this is a large jump, 4.0.0 to 5.9.0, can you confirm everything still works after the upgrade? I believe this module is used by GeoFence, or not?

Reply:
@aaime nope, currently it's used by GeoNode only (as far as I know)